flask_rebar/compat.py

```diff
@@ -1,15 +1,21 @@
+import abc
+import collections
 import sys
 import pkg_resources
 
 import marshmallow
 
-PY2 = int(sys.version_info[0]) == 2
+
+if int(sys.version_info[0]) == 2:
+    Mapping = collections.Mapping
+else:
+    Mapping = collections.abc.Mapping
 
 
-if PY2:
-    from collections import Mapping
+if sys.version_info >= (3, 4):
```

Comment:
I don't see ABC referenced anywhere - is this (still, or soon-to-be) needed? (Or some mystical Python trickery? :D )

Reply:
IMO, in modern Python, any base class should inherit from ABC.

Reply:
Oh, GitHub outsmarted me - the reference was hidden behind a "Load Diff" for a large file.  Carry on... :)